### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by kol-oss**
**KPI Year 1**
Refactoring of two programs:
- count.js
- reverse.js

### CODE REVIEW

File: Tasks/count.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
-    const value = obj[key];
-    if (typeof value === 'number') sum += value;
+const count = (obj) => {
+  let sum = 0;
+  const values = Object.values(obj);
+  values.map((value) => {
```
Comment (erotourtes) [inline Tasks/count.js:7]: The `map` is returning a new array, but you don't need it.

File: Tasks/reverse.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-    delete DATA[_];
-  }, 1000);
-  return DATA;
+const reverse = (obj) => {
+  const entries = Object.entries(obj);
+  const entriesRev = entries.map(
+    item => item.reverse()
+  );
+  
+  const result = Object.fromEntries(entriesRev);
```
Comment (erotourtes) [inline Tasks/reverse.js:10]: Think about returning the `result` directly

**Review by erotourtes — CHANGES_REQUESTED** [review]
There is something to improve

File: Tasks/count.js [hunk 3]
```diff
@@ -3,10 +3,10 @@
 
 const count = (obj) => {
```
Comment (BbIXOD) [inline Tasks/count.js:4]: Unexpected parentheses around single function argument `obj`

File: Tasks/count.js [hunk 4]
```diff
@@ -3,10 +3,10 @@
...
   let sum = 0;
-  const keys = Object.keys(obj);
-  keys.map((key) => {
-    const value = obj[key];
-    if (typeof value === 'number') sum += value;
+  const values = Object.values(obj);
+  values.map((value) => {
```
Comment (BbIXOD) [inline Tasks/count.js:7]: There are unnecessary parentheses too (`value`)

**Review by erotourtes — APPROVED** [review]
LGTM

**Comment by BbIXOD** [comment]
I think now all OK.

**Pull request merged**